src/client/src/orangehrmPerformancePlugin/pages/review.scss

```diff
@@ -33,3 +33,8 @@
     }
   }
 }
+
+.orangehrm-button{
```

Comment:
in the style block above it should do this. check why its not applying? 

```
@include oxd-respond-to('md') {
    margin-left: unset;
    max-width: unset;
    ::v-deep(.oxd-button--medium) {
      width: unset;
    }
  }
```

Reply:
sure will check